  CC    i810_io.o
In file included from i810_driver.c:74:
i830.h:65:19: error: sarea.h: No such file or directory


---
 src/i830.h |    2 ++
 1 file changed, 2 insertions(+)

Index: xf86-video-intel-2.8.0/src/i830.h
===================================================================
--- xf86-video-intel-2.8.0.orig/src/i830.h
+++ xf86-video-intel-2.8.0/src/i830.h
@@ -61,6 +61,7 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #include "xorg-server.h"
 #include <pciaccess.h>
 
+#ifdef XF86DRI
 #include "xf86drm.h"
 #include "sarea.h"
 #define _XF86DRI_SERVER_
@@ -69,6 +70,7 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #include "i830_dri.h"
 #include "intel_bufmgr.h"
 #include "i915_drm.h"
+#endif
 
 #include "uxa.h"
 Bool i830_uxa_init(ScreenPtr pScreen);
